include/mbgl/util/unique_any.hpp

```diff
@@ -1,17 +1,19 @@
 #pragma once
```

Comment:
Let's just replace `unique_any` with [this code](https://github.com/mapbox/mapbox-bindgen/blob/master/src/support/mapbox/bindgen/peer.hpp) from mapbox-bindgen. (For now we can just copy it.)